media/css/firefox/new/quantum/_other-platforms.scss

```diff
@@ -0,0 +1,202 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import '../../../pebbles/includes/lib';
+
+$color-quantum-blue: #0a84ff;
+
+/* -------------------------------------------------------------------------- */
+// Link & button styles for advanced install options & other languages.
+
+#other-platforms-modal-link {
+    @include font-size-small;
+    background: none;
+    border: none;
+    color: #fff;
+    cursor: pointer;
+    display: inline-block;
+    margin: 0;
+    padding: 0;
+
+    &:active,
+    &:focus,
+    &:hover {
+        text-decoration: underline;
+    }
+}
+
+/* -------------------------------------------------------------------------- */
+// Advanced install options modal styles.
+
+#other-platforms {
+    @include clearfix;
+    background: #fff;
+    color: #000;
+    margin-top: 20px;
+    text-align: center;
+
+    h4 {
+        color: #000;
+        font-weight: normal;
+        line-height: 1.3;
+        padding: 0 20px;
+
+        span {
+            display: block;
+            margin-top: 5px;
+            text-align: center;
+        }
+    }
+
+    .section-other-platforms {
+        margin-bottom: 20px;
+
+        ul {
+            padding: 0 20px;
+        }
+    }
+
+    .download-platform-list {
+        list-style: none;
+        margin: 20px auto;
+        text-align: center;
+
+        li {
+            display: block;
+            line-height: 2;
+            margin: 5px 0 0 -24px;
+        }
+
+        a:link,
+        a:visited {
+            @include font-size-small;
+            background: url('/media/img/firefox/new/quantum/down-arrow-blue.svg') left center no-repeat;
+            @include background-size(20px 16px);
+            color: $color-quantum-blue;
+            display: inline-block;
+            padding-left: 24px;
+        }
+    }
+
+    .other-platforms-mobile {
+        ul {
+            list-style: none;
+        }
+
+        li {
+            display: inline-block;
+            margin: 10px 0;
+        }
+    }
+
+    .section-current-platform {
+        margin-bottom: 40px;
+
+        h4 {
+            display: none;
+            margin-bottom: 20px;
+        }
+    }
+
+    .download-link .download-title {
+        color: #fff;
```

Comment:
I'm sure you're following design direction, but this white text on a neon green button is really hard to read due to lack of contrast. Can this be a place where we just do what's right?

Reply:
Yes, totally. Let's just revert this one to a regular button.